lib/r10k/module/forge.rb

```diff
@@ -44,12 +44,11 @@ def initialize(title, dirname, opts, environment=nil)
         :source  => ::R10K::Util::Setopts::Ignore,
         :type    => ::R10K::Util::Setopts::Ignore,
       })
-    else
-      @expected_version = opts || current_version || :latest
     end
 
     @metadata_file = R10K::Module::MetadataFile.new(path + 'metadata.json')
     @metadata = @metadata_file.read
+    @expected_version = opts || current_version || :latest
```

Comment:
Do we want to be setting `@expected_version` to `opts` when `opts` _is_ a Hash?

Reply:
I'm not sure, I broke a test, so something needs to change

Reply:
Seems like we do not, the third option used to just be the expected version itself, so it made sense to use that as a first try. Maybe we have to test for that?

Reply:
Alright I just rearranged but in a different way, it passed specs locally for me...